app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def welcome_email
+    @user = params[:user]
+    @url  = 'http://example.com/login'
```

Comment:
Em trỏ đúng vào trang login của app nha em, ví dụ như muốn vào trang show toàn bộ users thì là `users_path`, kiểu vậy